### ISSUE TITLE
BSOD when slicing

### ISSUE THREAD

**Issue by Kobmar96** [bambulab/BambuStudio#3354]
### Bambu Studio Version

1.8.2.56

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Additional system information

CPU: i9 14900k
RAM: DDR5 (Corsair Dominator Titanium)
GPU: RTX 4090

### Printer

Bambu Lab X1C

### How to reproduce

Slice a complex project. I heard from different people on Reddit, having the same problem, running also on i9 14900k and DDR5 RAM.
There is a relationship to issue https://github.com/SoftFever/OrcaSlicer/issues/2921 and https://github.com/SoftFever/OrcaSlicer/issues/3375

### Actual results

Either BambuStudio will just close randomly, or the PC will freeze completely and give a BSOD "Kernel Power"

### Expected results

Need to restart the PC.

### Project file & Debug log uploads

[Logs.zip](https://github.com/bambulab/BambuStudio/files/13899298/Logs.zip)
[Project.zip](https://github.com/bambulab/BambuStudio/files/13899299/Project.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by bambu123** [comment]
>	BambuStudio.dll!__report_gsfailure(unsigned __int64 stack_cookie) 行 220	C	非用户代码。已加载符号。
 	BambuStudio.dll!Slic3r::ClipperLib::Clipper::ProcessEdgesAtTopOfScanbeam(const int topY) 行 2681	C++	已加载符号。
 	BambuStudio.dll!Slic3r::ClipperLib::Clipper::ExecuteInternal() 行 1134	C++	已加载符号。
 	BambuStudio.dll!Slic3r::ClipperLib::Clipper::Execute(Slic3r::ClipperLib::ClipType clipType, std::vector<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>,std::allocator<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>>> & solution, Slic3r::ClipperLib::PolyFillType subjFillType, Slic3r::ClipperLib::PolyFillType clipFillType) 行 1091	C++	已加载符号。
 	BambuStudio.dll!Slic3r::clipper_union<std::vector<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>,std::allocator<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>>>,std::vector<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>,std::allocator<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>>> &>(std::vector<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>,std::allocator<std::vector<Slic3r::Point,std::allocator<Slic3r::Point>>>> & subject, const Slic3r::ClipperLib::PolyFillType fillType) 行 348	C++	已加载符号。
 	BambuStudio.dll!Slic3r::expolygons_offset<std::vector<Slic3r::ExPolygon,std::allocator<Slic3r::ExPolygon>>>(const std::vector<Slic3r::ExPolygon,std::allocator<Slic3r::ExPolygon>> & expolygons, const float delta, Slic3r::ClipperLib::JoinType joinType, double miterLimit) 行 530	C++	已加载符号。
 	BambuStudio.dll!Slic3r::offset(const std::vector<Slic3r::ExPolygon,std::allocator<Slic3r::ExPolygon>> & expolygons, const float delta, Slic3r::ClipperLib::JoinType joinType, double miterLimit) 行 549	C++	已加载符号。
 	BambuStudio.dll!Slic3r::PerimeterGenerator::generate_lower_polygons_series(float width) 行 1685	C++	已加载符号。
 	BambuStudio.dll!Slic3r::PerimeterGenerator::process_classic() 行 847	C++	已加载符号。
 	BambuStudio.dll!Slic3r::LayerRegion::make_perimeters(const Slic3r::SurfaceCollection & slices, Slic3r::SurfaceCollection * fill_surfaces, std::vector<Slic3r::ExPolygon,std::allocator<Slic3r::ExPolygon>> * fill_no_overlap) 行 112	C++	已加载符号。
 	BambuStudio.dll!Slic3r::Layer::make_perimeters() 行 194	C++	已加载符号。
 	[内联框架] BambuStudio.dll!Slic3r::PrintObject::make_perimeters::__l2::<lambda_df1d3487ef7218a95c0f8a8e19180a17>::operator()(const tbb::detail::d1::blocked_range<unsigned __int64> &) 行 226	C++	已加载符号。
 	[内联框架] BambuStudio.dll!tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<unsigned __int64>,<lambda_df1d3487ef7218a95c0f8a8e19180a17>,tbb::detail::d1::auto_partitioner const>::run_body(tbb::detail::d1::blocked_range<unsigned __int64> & r) 行 119	C++	已加载符号。
 	BambuStudio.dll!tbb::detail::d1::dynamic_grainsize_mode<tbb::detail::d1::adaptive_mode<tbb::detail::d1::auto_partition_type>>::work_balance<tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<unsigned __int64>,<lambda_df1d3487ef7218a95c0f8a8e19180a17>,tbb::detail::d1::auto_partitioner const>,tbb::detail::d1::blocked_range<unsigned __int64>>(tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<unsigned __int64>,<lambda_df1d3487ef7218a95c0f8a8e19180a17>,tbb::detail::d1::auto_partitioner const> & start, tbb::detail::d1::blocked_range<unsigned __int64> & range, tbb::detail::d1::execution_data & ed) 行 462	C++	已加载符号。
 	[内联框架] BambuStudio.dll!tbb::detail::d1::partition_type_base<tbb::detail::d1::auto_partition_type>::execute(tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<unsigned __int64>,<lambda_df1d3487ef7218a95c0f8a8e19180a17>,tbb::detail::d1::auto_partitioner const> &) 行 288	C++	已加载符号。
 	BambuStudio.dll!tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<unsigned __int64>,<lambda_df1d3487ef7218a95c0f8a8e19180a17>,tbb::detail::d1::auto_partitioner const>::execute(tbb::detail::d1::execution_data & ed) 行 173	C++	已加载符号。
 	BambuStudio.dll!tbb::detail::r1::task_dispatcher::local_wait_for_all<0,class tbb::detail::r1::outermost_worker_waiter>(class tbb::detail::d1::task *,class tbb::detail::r1::outermost_worker_waiter &)	C++	非用户代码。已加载符号。
 	BambuStudio.dll!tbb::detail::r1::arena::process(class tbb::detail::r1::thread_data &)	C++	非用户代码。已加载符号。
 	BambuStudio.dll!tbb::detail::r1::market::process(class rml::job &)	C++	非用户代码。已加载符号。
 	BambuStudio.dll!tbb::detail::r1::rml::private_worker::run(void)	C++	非用户代码。已加载符号。
 	BambuStudio.dll!tbb::detail::r1::rml::private_worker::thread_routine(void *)	C++	非用户代码。已加载符号。

@QingZhangBambu

**Comment by QingZhangBambu** [comment]
@Kobmar96 
holle, there is a method that some people said is helpful, may you try it and check if it is useful?.
For Win 11, here is how to set the affinity :
1- Open Task Manager (CTRL+SHIFT+ESCAPE)
2- Go to Details
3- find bambu-studio.exe and right click to chose "Affinity"
image
4- From the popup, it is possible to choose which thread(s) to use. For example, on my i9-13900K, I've got CPU0 to CPU31 (I've 8 performance cores with 2 threads, and 16 efficiency cores -> 32 "CPUs" listed)
image
And interestingly enough, when I slice without setting affinity (so, Bambu Studio uses all threads available) -> it crashes.
When I first set affinity to, for example, CPU0 to CPU3, or CPU0 to CPU7 -> slicing goes through successfully.
